CHANGELOG.md

```diff
@@ -28,6 +28,126 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+## [v7.0.0-beta5](https://github.com/shopsys/shopsys/compare/v7p.0.0-beta4...v7.0.0-beta5) - 2019-01-17
+
+### [shopsys/coding-standards]
+
+#### Changed
+
+- [#707] order of imports and sort algorithm is now explicitly set
+
+### [shopsys/framework]
+
+#### Added
+
+- [#619] Simplify adding new form groups
+- [#723] Copy categories to variant if main variant is edited or variant is added
+
+#### Changed
+
+- [#688] renamed AdvancedSearchFacade to AdvancedSearchProductFacade
+- [#650] change visibility in extendable classes according to our docs
+- [#692] redirectController is now registered as a public container service
+- [#624] ImageUploadType now can be used for uploading specific image types
+- [#690] refactored checking duplicated e-mail to User constructor
+- [#712] Stop generating url for unsafe route 'fp_js_form_validator.check_unique_entity'
```

Comment:
sounds like a *fix* to me

Reply:
I would like to thank pk16011990 for this one as he was the original author of #298